schemas/com.snowplowanalytics.snowplow.storage/redshift_config/jsonschema/3-0-0

```diff
@@ -0,0 +1,250 @@
+                            "type": "string",
+                            "minLength": 3,
+                            "maxLength": 255
+                        }
+                    },
+                    "required": ["tableName"]
+                }
+            },
+            "required": ["amazonDynamoDb"]
+        },
+        "jdbc": {
+            "type": "object",
+            "properties": {
+                "BlockingRowsMode": {
+                    "description": "The number of rows to hold in memory",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "DisableIsValidQuery": {
+                    "description": "Disable detection JDBC driver no longer holding a valid database connection",
+                    "type": "boolean"
+                },
+                "DSILogLevel": {
+                    "description": "Amount of detail included in log file",
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 6
+                },
+                "FilterLevel": {
+                    "description": "The minimum severity level of a message that the client processes",
+                    "enum": ["DEBUG", "INFO", "NOTICE", "WARNING", "LOG", "ERROR"]
+                },
+                "loginTimeout": {
+                    "description": "The number of seconds to wait before timing out when connecting to the server",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "loglevel": {
+                    "description": "The amount of logging information output by the driver",
+                    "type": ["integer", "null"],
+                    "minimum": 0,
+                    "maximum": 2
+                },
+                "socketTimeout": {
+                    "description": "The number of seconds to wait during socket read operations before timing out",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "ssl": {
+                    "description": "Whether to use an SSL connection",
+                    "type": "boolean"
+                },
+                "sslMode": {
+                    "description": "Determines how to handle server certificate verification",
+                    "type": ["string", "null"],
+                    "enum": ["verify-ca", "verify-full"]
+                },
+                "sslRootCert": {
+                    "description": "The full path of a .pem or .crt file containing the trust root Certificate Authority",
+                    "type": ["string", "null"],
+                    "maxLength": 1024
+                },
+                "tcpKeepAlive": {
+                    "description": "A value that determines whether TCP keepalives are enabled",
+                    "type": "boolean"
+                },
+                "TCPKeepAliveMinutes": {
```

Comment:
Why is the capitalisation inconsistent with `tcpKeepAlive`

Reply:
All JDBC options are written as they're on driver's documentation.